src/resolver.rs

```diff
@@ -0,0 +1,29 @@
+//! Entity resolver module
+
+use crate::events::BytesText;
+
+/// Used to resolve unknown entities while parsing
+pub trait EntityResolver {
```

Comment:
Please add an example here as suggested in [my comment](https://github.com/tafia/quick-xml/issues/581#issuecomment-1488823175)